rosidl_runtime_rs/src/string.rs

```diff
@@ -224,6 +225,42 @@ macro_rules! string_impl {
             }
         }
 
+        impl From<&std::string::String> for $string {
+            fn from(s: &std::string::String) -> Self {
+                Self::from(s.as_str())
+            }
+        }
+
+        impl FromIterator<char> for $string {
+            fn from_iter<I: IntoIterator<Item = char>>(iter: I) -> Self {
+                let mut buf = <$string>::default();
+                buf.extend(iter);
+                buf
+            }
+        }
+
+        impl<'a> FromIterator<&'a char> for $string {
+            fn from_iter<I: IntoIterator<Item = &'a char>>(iter: I) -> Self {
+                let mut buf = <$string>::default();
+                buf.extend(iter);
+                buf
+            }
+        }
+
+        impl Extend<char> for $string {
```

Comment:
Small nitpick: I think it's nice to have trait impls in alphabetical order for readability. Not important though.

Reply:
Sure thing, alphabetized all the trait impl's in this module.